vncviewer/Viewport.cxx

```diff
@@ -1197,6 +1204,19 @@ void Viewport::handleAltGrTimeout(void *data)
   self->altGrArmed = false;
   self->handleKeyPress(0x1d, XK_Control_L);
 }
+
+void Viewport::resolveAltGrDetection(void *data, bool isAltGrSequence)
+{
+  Viewport *self = (Viewport *)data;
```

Comment:
No need for this. Make it a normal, proper method.

Reply:
Fixed in 944c527082786ee293de91ae76ca70fab1e6af81